coalib/misc/BuildManPage.py

```diff
@@ -27,7 +27,7 @@ class BuildManPage(Command):
     You can then use the following setup command to produce a man page::
 
         $ python setup.py build_manpage --output=coala.1 \
-            --parser=coalib.parsing.DefaultArgParser:default_arg_parser
+            --parser=coalib.parsing.Defaultargparse.ArgumentParser:default_arg_parser
```

Comment:
Please don't change `DefaultArgParser`. That's really an internal coala module, containing the `default_arg_parser` function, that creates `argparse.ArgumentParser` instances